pkg/cmd/addon/enable/exec.go

```diff
@@ -106,7 +106,22 @@ func (o *Options) runWithClient(clusterClient clusterclientset.Interface,
 		}
 	}
 
+	cmas, err := addonClient.AddonV1alpha1().ClusterManagementAddOns().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
 	for _, addon := range addons {
+		cmaExist := false
+		for _, cma := range cmas.Items {
```

Comment:
can we just get cma here instead of list? It will require less permission.

Reply:
PTAL